torch/utils/cpp_extension.py

```diff
@@ -139,12 +152,19 @@ def _define_torch_extension_name(self, extension):
 
 def CppExtension(name, sources, *args, **kwargs):
     '''
-    Create a setuptools.Extension instance for C++.
+    Create a `setuptools.Extension` for C++.
 
     Convenience method that creates a `setuptools.Extension` with the bare
     minimum (but often sufficient) arguments to build a C++ extension.
 
-    All arguments are forwarded to the setuptools.Extension constructor.
+    All arguments are forwarded to the `setuptools.Extension` constructor.
+
+    Example:
+        >>> from torch.utils.cpp_extension import CppExtension
+        >>> CppExtension(
+                name='extension',
+                sources=['extension.cpp'],
+                extra_compile_args=['-g'])
```

Comment:
This is not really an example of a real use, right? I don't think it will be very helpful if shown out of context

Reply:
Ok. I'll add the setuptools stuff around it